src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -577,6 +577,10 @@ void launchStreams(List<String> streamIds) {
     }
 
     Object[] checkUploadStream(Object[] args, List<String> streamIds) {
```

Comment:
Maybe this should be renamed something `checkUploadStream` now that it's doing more than setting up the streamMap. Maybe `processArgs` or `normalizeArgs`

Reply:
I don't disagree, but I've already had to triage an rerun checks for non related failures 3 times on this PR so I don't want to risk it 😅